core/src/main/java/hudson/cli/handlers/ViewOptionHandler.java

```diff
@@ -73,10 +75,25 @@ public int parseArguments(Parameters params) throws CmdLineException {
         return 1;
     }
 
-    private View getView(String name) throws CmdLineException {
+    /**
+     *
+     * Gets a view by its name
```

Comment:
Makes sense to mention that personal user views are not supported (could be a topic for the improvement)